docs/spec.md

```diff
@@ -6285,6 +6324,33 @@ and [slicing](https://docs.python.org/3/reference/expressions.html#slicings)
 notations from Python are available to index into tensors, quantized tensors
 and tuples.
 
+* `convert_or_quantize(x: Value, destination_type: Type) -> Value` is defined on
+tensors and returns the converted value of `x` based on the `type(x)` and
+`destination_type` as follows:
+
+```python
+def convert_or_quantize(x: Value, destination_type: Type) -> Value:
```

Comment:
For me, 'convert_or_quantize' sounds like it will convert or quantize, and never dequantize, but it actually does dequantize. I'm not sure if 'convert_or_quantize_or_dequantize' is too verbose, but can we somoehow reflect the dequantizing behavior on its name?

Reply:
I agree. Let me get back on that.

Reply:
Sounds good to me!